assets/js/base/components/shipping-rates-control/style.scss

```diff
@@ -1,15 +1,17 @@
-.wc-block-shipping-rates-control__no-results {
-	border-bottom: 1px solid $core-grey-light-600;
-	margin-bottom: 0;
-	padding-bottom: $gap-small;
+.wc-block-shipping-rates-control__package + .wc-block-shipping-rates-control__package {
```

Comment:
It's not clear to me why this is being done (could be a learning thing for me).

Reply:
one there is more than one package, style all subsequence packages (2, 3, 4...)

Reply:
could also be solved by
`.wc-block-shipping-rates-control__package:not(:first-of-type)`

Reply:
Right, it's the [adjacent selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator) so it selects all `.wc-block-shipping-rates-control__package` elements which are after another `.wc-block-shipping-rates-control__package` element.

But I like @senadir's solution more, so I will change this. :smile:

Reply:
Done in 2082e3dd7e6f808e9e7cf4d8c6679bd2dcdbcbb5. :heavy_check_mark: